packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -573,6 +573,8 @@
       <name>quantic_InclusionFilter_zero</name>
     </customLabels>
     <customLabels>
+      <label>Ouvre dans un sous-onglet de la console Salesforce</label>
+      <name>quantic_OpensInSalesforceSubTab</name>
```

Comment:
I see that there was two `customLabels ` missing, that may have occurred after a conflict after pulling from master, please confirm that all the tags are close properly in this file.
```suggestion
      <name>quantic_OpensInSalesforceSubTab</name>
    </customLabels>
    <customLabels>
```

Reply:
mmm odd, I will repush and see for conflicts, seems fine on my end for the closures